.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 /.env
 .vagrant
 vault.key
+cgus-data
+*.lock
```

Comment:
This was meant to ignore `yarn.lock` files. This may be best reverted, depends on your setup and preferences, since you keep track of `package-lock.json` in the repo.

Reply:
Yes, the NPM lockfile [is intended to be committed into source repositories](https://docs.npmjs.com/configuring-npm/package-lock-json) 🙂 
This is what enables consistent deployments.

Reply:
We wouldn't want to commit a Yarn lockfile though, so this line can stay there. Maybe it should be rewritten to `yarn.lock` though.

Reply:
Done.